src/Shared/BuildEnvironmentHelper.cs

```diff
@@ -599,7 +599,11 @@ public BuildEnvironment(BuildEnvironmentMode mode, string currentMSBuildExePath,
 
                 MSBuildToolsDirectory32 = MSBuildToolsDirectoryRoot;
                 MSBuildToolsDirectory64 = existsCheck(potentialAmd64FromX86) ? Path.Combine(MSBuildToolsDirectoryRoot, "amd64") : CurrentMSBuildToolsDirectory;
+#if RUNTIME_TYPE_NETCORE
+                MSBuildToolsDirectoryArm64 = existsCheck(potentialARM64FromX86) ? Path.Combine(MSBuildToolsDirectoryRoot, "arm64") : CurrentMSBuildToolsDirectory;
```

Comment:
Seem reasonable?

```suggestion
                // Fall back to "current" for any architecture since .NET SDK doesn't
                // support cross-arch task invocations.
                MSBuildToolsDirectoryArm64 = existsCheck(potentialARM64FromX86) ? Path.Combine(MSBuildToolsDirectoryRoot, "arm64") : CurrentMSBuildToolsDirectory;
```